ui/src/app/stores/auth/auth.reducers.ts

```diff
@@ -13,13 +13,13 @@
  * limitations under the License.
  */
 
-import * as AuthActions from "./auth.actions";
+import * as AuthActions from './auth.actions';
 import {localStorageKeys} from '../../constants/localStorage.constants';
 
 export interface State {
-  username: string,
-  isAuthenticated: boolean,
-  authenticationFailed: boolean
+  username: string;
```

Comment:
What's bad about using commas instead of semicolons here?

Reply:
IDE: properties should be separated by semicolon, not commas.

Reply:
ok